.github/workflows/deploy.yml

```
@@ -20,7 +22,9 @@ jobs:
             python_version: "3.7"
           - name: python3.6
             python_version: "3.6"
-          - name: python3.9-slim
+          - name: python3.10-slim
+            python_version: "3.9"
+          - name: python3.10-slim
             python_version: "3.9"
```

```suggestion
          - name: python3.10-slim
            python_version: "3.10"
          - name: python3.9-slim
            python_version: "3.9"
```

Maybe I'm missing my morning caffeine, but I think this is what you meant?

---

That's right, thank you!